### DISCUSSION TITLE
Rebuilding a forum from backup (Can someone explain the process?)

### DISCUSSION THREAD

**Discussion by gordsellar** [e107inc/e107#5324] [Q&A Installation] [answered]
Hi there, 

I was given a copy of a backup for an old forum that was hosted with a version of e107. The backup seems to date back to 2018, and contains multiple tar.gz files, a .tgz file, and some .tar files.  

I was trying to figure out how to rebuild a site from that backup, but I can't really seem to find any helpful guide for doing that. I feel like if I could find a step-by-step guide I could probably figure it out, but I can't even really find that. 

Any suggestions?

**Comment by Jimmi08** [comment]
Post screenshots of file names (content of backup) 

Be sure to use older PHP version (f.e PHP 5 for testing, max PHP 7) and after you unpack the files pack, you will see what version was used.  
Or if you have a backup of all site or only of forum part.

**Reply by gordsellar** [reply]
This us the file list for the backup files I got. The site isn't online and hasn't been for years. 

![Screenshot 2024-08-23 at 12 21 25 PM](https://github.com/user-attachments/assets/0eb331b8-aa7d-4652-985f-bcf7de1b8e7f)

I assume it's one of the last two tar.gz files I need to use, since they're both the largest. (The latter is slightly larger, about 20 MB more, which makes me think it's a backup of the site files.)

I don't know why they're named the way they are, the site went offline (IIRC) in 2017.

**Reply by Jimmi08** [reply]
Your site was paranoia-live.net, correct?
it used e107 version 1, so at first you could get a server with PHP 5 to get it running.  
From looking at navigation, it uses old plugins like content, faq, links...   there is no update of them, only manually copy content. It is too late for update.

If you don't need this content or it is enough to copy it from cached version on internet, you don't need to install old system.  (that site was fairly famous, it is pretty well cached)

![image](https://github.com/user-attachments/assets/98b7bbb4-eb3b-4ace-938e-36336372bee9)

It looks like they used PHPbb forum, not e107 one.  Until 2012/13 it is clear PHPBB forum (separate installation),  until 2015 it looks like updated version, not sure, layout is broken. 

See - it is PHPBB:
![image](https://github.com/user-attachments/assets/7df6cb29-347d-449b-8bda-5a41572b9ad3)

This is not e107 forum either (2014/2015) - maybe updated PHPbb forum, links are different.  But definitely not e107. 
![image](https://github.com/user-attachments/assets/672fbc76-15d2-4f98-a037-162af2ad51b9)

Site ended in 2016.

So you have separate installations of 2 different CMS.  Maybe this explains so many files in your backup. 
So what do you want to save (revive)?

**Comment by gordsellar** [comment]
Hi there, 

I wasn't running the site, but I was given a backup of the original site by one of the people who used to admin it, back when there was a group of people trying to resurrect it. Maybe it was PhoBB, though I was told it had used e107, and when I looked through one of the tarballs I found files that mention e107, so that's really odd. 

I was just trying to figure out whether it's possible to resurrect the site on a subdomain of my own website, so I could look around. Last time I tried to find an archive wayback machine, a lot of threads weren't fully accessible. I wouldn't be against actually resurrecting the forum at some point—though the people I knew who were working on it gave up on that project—but I was mostly just hoping to look around and see what I'd missed, since I never heard about the site until after it went offline.

**Reply by Jimmi08** [reply]
If you can set PHP 5 for your subdomain, it is possible. Look what is inside paranoia-live.net folder. if site files, look at e107_config.php for actual name of database (because you have there more sql files). If not, unpack some backups to see what is there.  Then go forum folder (or phpbb backup files) and look for config and database name.  Either you have 2 databases or 1 database with 2 prefixes.

**Reply by gordsellar** [reply]
Okay, I'll try that. Thanks!

**Reply by Jimmi08** [reply]
if you are interested in articles on main site, there is content plugin used, look for content.php in site root (so you know it is e107 copy). In database it is "pcontent" table.   One table for everything.

**Reply by gordsellar** [reply]
Thanks!

**Comment by gordsellar** [comment]
I also noticed that *someone* has put up an "I'm back" notice on the Paranoia-live.net address, so maybe this effort is pointless, who knows.

**Reply by Jimmi08** [reply]
You are right, Wayback noticed this in March 2024.

**Reply by gordsellar** [reply]
> You are right, Wayback noticed this in March 2024.

And a little more searching online reveals it's run by the person who gave me the backup copy, still working on getting it restored many years later. (I'd assumed he'd given up on the project, but apparently not.) I've left him a message to see whether he's continuing work on the project.

**Reply by tgtje** [reply, marked as answer]
Could be done if everything is present, however will take a lot of effort.
Suppose you are running windoos : it will not support every old local files you need, so best suited was a virtual environment.
If you up to it find a wamp or xamp server 32 bit version (wmap example 2.2) where php 5.4.3 is highest!!!.
unpack ALL files and as @Jimmi08 mentionded find the config file. it will give a glance how the website structure itself is presented (by folders).The rest of folders and 'single files' come in root of the site.
Now it is unclear if for phpbb a plugin is used or a single installment. (either a folder called phpbb or inside the plugins dir.
Now as mentioned config has the database file, import that one in phpadmin sql (likely a mysql).
 All the files are in a 'root' with the websites name (eg domainname alike). 
 YOu might get issues with the size of sql file as older versions of phpadmin only accept small sizes, You could (if possible) adjust some settings to enlarge the 'upload/import size) OR manual split and copy paste the parts into the database using insert. (just do not copy identical lines), can be done in good editor like Notepad++ or alike, keeping the splitparts below the max size.
 The next will be the password for access the site (imported to database). You might need to change it (in dbase) but ALSO in the config file so site and database can communicate. Best to change the password AFTER you have tested the site 'appears'at all.
 Suppose it should function, the next steps would be gradually update (helas ALL) to newer versions untill you are up to date.
 Updates for nearly all can be found, just remember each time you try to upgrade, make backups from databse, and files.
 All is not done in a single day..
 If you are familiar with a lot of stuff you COULd use a new wamp version wamp (64bit v3), add php versions of max 5.4.3 but you need to have an older phpmyadmin (version 4.0.10 or alike) to commuicate. I succeeded with such using a v 0.7.4 version  dated 2006 running on a W11 machine.
 images of similar structure 1 and version 0.7.4 (2)
 
![rppt](https://github.com/user-attachments/assets/9e8d679f-1c24-45a3-b989-face5c86f956)

![rppt2](https://github.com/user-attachments/assets/93e1e78a-0f43-4af4-be1d-c47cb42bd5b1)

success

**Reply by tgtje** [reply]
just quick sidenote.. using wayback... was domain name in another hands? as some screenshots display a chinese site ??

**Reply by gordsellar** [reply]
Thanks, that's a very detailed response and it'll help if I do end up needing to do this myself. 

I think the domain name did fall into someone else's hands for a while. I don't have the domain myself even now. (But I know the person who does.)

**Reply by tgtje** [reply]
YW, just **remember. it is still local**.. for live you just need to upload all, but adjust config file !!  to the new server settings.
Proces is basically similar. export dbase, upload files, adjust config and if done, the site should run (keep on eye out for the access file. )
